src/main/java/com/iexec/worker/feign/LoginService.java

```diff
@@ -48,21 +45,30 @@ public String getToken() {
         return jwtToken;
```

Comment:
We should probably do something if `jwtToken` is null or empty. Either we automatically log in, either we throw an exception.

Reply:
Following our discussion, we won't touch this method for the moment.
Still, we have identified thread-safety concerns in our implementations which will need to be addressed in the future.